### ISSUE TITLE
Error occurred while enabling EnderContainers v2.1.0 (Is it up to date?) java.lang.NullPointerException: null

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#23]
```
[23:45:58 INFO]: [EnderContainers] Enabling EnderContainers v2.1.0
[23:45:58 WARN]: [EnderContainers] class fr.utarwyn.endercontainers.command.CommandManager instance is null!
[23:45:58 ERROR]: Error occurred while enabling EnderContainers v2.1.0 (Is it up to date?)
java.lang.NullPointerException: null
        at fr.utarwyn.endercontainers.dependency.EssentialsDependency.onEnable(EssentialsDependency.java:25) ~[?:?]
        at fr.utarwyn.endercontainers.dependency.DependenciesManager.registerDependency(DependenciesManager.java:132) ~[?:?]
        at fr.utarwyn.endercontainers.dependency.DependenciesManager.loadDependencies(DependenciesManager.java:95) ~[?:?]
        at fr.utarwyn.endercontainers.dependency.DependenciesManager.load(DependenciesManager.java:42) ~[?:?]
        at fr.utarwyn.endercontainers.Managers.registerManager(Managers.java:32) ~[?:?]
        at fr.utarwyn.endercontainers.EnderContainers.onEnable(EnderContainers.java:55) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263) ~[patched_1.14.3.jar:git-Paper-112]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338) ~[patched_1.14.3.jar:git-Paper-112]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:419) ~[patched_1.14.3.jar:git-Paper-112]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugin(CraftServer.java:464) ~[patched_1.14.3.jar:git-Paper-112]
        at org.bukkit.craftbukkit.v1_14_R1.CraftServer.enablePlugins(CraftServer.java:378) ~[patched_1.14.3.jar:git-Paper-112]
        at net.minecraft.server.v1_14_R1.MinecraftServer.a(MinecraftServer.java:464) ~[patched_1.14.3.jar:git-Paper-112]        at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:282) ~[patched_1.14.3.jar:git-Paper-112]
        at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:859) ~[patched_1.14.3.jar:git-Paper-112]
        at java.lang.Thread.run(Thread.java:835) [?:?]
[23:45:58 INFO]: [EnderContainers] Disabling EnderContainers v2.1.0
```

**Comment by utarwyn** [comment]
What a strange bug... I fix that right now, sorry for that!

**Comment by utarwyn** [comment]
You can find the commit which fixes this error here : 4542bdbc719271380babc7270a5da483005c02c8
Is it good for you? Thank you for the feedback about that! 😄

**Comment by andris155** [comment]
Works perfectly. Thanks.
